Modules/TPC/src/PID.cxx

```diff
@@ -97,7 +103,45 @@ void PID::monitorData(o2::framework::ProcessingContext& ctx)
 
 void PID::endOfCycle()
 {
+  // ===| Fitting Histogram for separation Power |============================================================
+  for (auto const& pair : mQCPID.getMapOfHisto()) {
+    for (auto& hist : pair.second) {
+      //      if (std::static_cast<string>(pair.first).compare("hdEdxMaxMIP") == 0) {
+      if (pair.first.compare("hdEdxMaxMIP") == 0) {
+        mTrendingParameters.clear();
+        // Define fitting function: sum of two Gaussians with an offset
+        TF1* fitFunc = new TF1("fitFunc", "[0]*exp(-0.5*((x-[1])/[2])^2) + [3]*exp(-0.5*((x-[4])/[5])^2)", 0, 100);
+
+        // Set initial parameters for the fit
+        fitFunc->SetParameter(0, 3000); // Amplitude of the first Gaussian
+        fitFunc->SetParameter(1, 50);   // Mean of the first Gaussian
+        fitFunc->SetParLimits(1, 45, 55);
+        fitFunc->SetParameter(2, 2);   // Sigma of the first Gaussian
+        fitFunc->SetParameter(3, 100); // Amplitude of the second Gaussian
+        fitFunc->SetParameter(4, 75);  // Mean of the second Gaussian
+        fitFunc->SetParLimits(4, 60, 90);
+        fitFunc->SetParameter(5, 10); // Sigma of the second Gaussian
+
```

Comment:
Why is this inside the loop? The same fit function is used. So either:
- put it on the stack, or
- move it outside the and make it a `unique_ptr`.